api/v1alpha1/imagemap_types.go

```diff
@@ -76,6 +76,7 @@ const (
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:webhook:verbs=create;update,path=/image-mapper,mutating=true,failurePolicy=fail,groups="",resources=pods,versions=v1,name=image-mapper.imagerelocation.pivotal.io
```

Comment:
Typically this is where you'd define a webhook for the ImageMap type, not any arbitrary webhook.

Reply:
Yeah, I had Glyn put it here just to get things working 😄

Reply:
Not really sure where to put this, but I'll stick it somewhere in the webhook code for now.